compiler-core/src/analyse.rs

```diff
@@ -93,6 +93,16 @@ pub enum TargetSupport {
     NotEnforced,
 }
 
+impl TargetSupport {
+    /// Returns `true` if the target support is [`Enforced`].
+    ///
+    /// [`Enforced`]: TargetSupport::Enforced
+    #[must_use]
+    pub fn is_enforced(&self) -> bool {
+        matches!(self, Self::Enforced)
```

Comment:
I remember an issue saying we should drop all uses of `matches!` https://github.com/gleam-lang/gleam/issues/1972

Reply:
Oh yes, thank you. Auto-generated code!